plugins/woocommerce/client/admin/client/analytics/components/report-chart/index.js

```diff
@@ -349,7 +349,7 @@ export default compose(
 		).getSetting( 'wc_admin', 'wcAdminSettings' );
 
 		/* eslint @wordpress/no-unused-vars-before-return: "off" */
-		const reportStoreSelector = select( REPORTS_STORE_NAME );
+		const reportStoreSelector = select( reportsStore );
```

Comment:
Maybe moving this line closer to where the variable will be used to eliminate the eslint off annotation.